x/auth/signing/sign_mode_handler.go

```diff
@@ -1,69 +1,26 @@
 package signing
 
 import (
-	"context"
+	apisigning "cosmossdk.io/api/cosmos/tx/signing/v1beta1"
+	"cosmossdk.io/x/tx/signing"
 
-	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/types/tx/signing"
 )
 
 // SignModeHandler defines a interface to be implemented by types which will handle
 // SignMode's by generating sign bytes from a Tx and SignerData
 type SignModeHandler interface {
 	// DefaultMode is the default mode that is to be used with this handler if no
 	// other mode is specified. This can be useful for testing and CLI usage
-	DefaultMode() signing.SignMode
+	DefaultMode() apisigning.SignMode
 
 	// Modes is the list of modes supporting by this handler
-	Modes() []signing.SignMode
+	Modes() []apisigning.SignMode
 
 	// GetSignBytes returns the sign bytes for the provided SignMode, SignerData and Tx,
 	// or an error
-	GetSignBytes(mode signing.SignMode, data SignerData, tx sdk.Tx) ([]byte, error)
+	GetSignBytes(mode apisigning.SignMode, data signing.SignerData, tx sdk.Tx) ([]byte, error)
```

Comment:
Looking at this interface, how much cleaner are we making things with this change?  We still have types split across two `signing` packages

Reply:
This brings it closer to the types from x/tx.